content/techniques/sql.md

```diff
@@ -576,6 +576,21 @@ export class AppModule {}
 
 > warning **Notice** If you don't set the `name` for a connection, its name is set to `default`. Please note that you shouldn't have multiple connections without a name, or with the same name, otherwise they will get overridden.
 
+> warning **Notice** If you are using `TypeOrmModule.forRootAsync`, you have to set connection name outside `useFactory`. For example:
+> ```typescript
+> TypeOrmModule.forRootAsync({
+>   name: 'albumsConnection',
+>   useFactory: (config: ConfigService) =>
+>     ({
+>       ...config.get('db'),
+>       host: 'album_db_host',
+>       entities: [Album],
+>     } as TypeOrmModuleOptions),
+>   inject: [ConfigService],
```

Comment:
```suggestion
>   useFactory: (config: ConfigService) =>
>     ({
>       ...config.get('db'),
>       host: 'album_db_host',
>       entities: [Album],
>     } as TypeOrmModuleOptions),
>   inject: [ConfigService],
```

Can we make this example less "specific"? (drop the use of `ConfigService` etc, basically get rid of irrelevant code blocks)